docs/root/configuration/http_filters/fault_filter.rst

```diff
@@ -71,6 +71,12 @@ fault.http.max_active_faults
   cause resource constraint issues. If not specified, the :ref:`max_active_faults
   <envoy_api_field_config.filter.http.fault.v2.HTTPFault.max_active_faults>` setting will be used.
 
+fault.http.rate_limit.response
```

Comment:
rate_limit.percentage? because this is controlling the percentage of requests and not the rate itself

Reply:
Sure will change.